src/main/java/com/nokia/uwr/callsysinitializer/CallSysInitializerImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.nokia.uwr.callsysinitializer;
+
+import com.nokia.uwr.connectionmanager.ConnectionManager;
+import com.nokia.uwr.model.BTS;
+import com.nokia.uwr.model.UEMeasurement;
+import lombok.RequiredArgsConstructor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class CallSysInitializerImpl implements CallSysInitializer {
+
+    private final ConnectionManager connectionManager;
+    private static final Logger LOGGER = LogManager.getLogger(CallSysInitializerImpl.class);
+
+    @Override
+    public void initializeAssignments(List<BTS> btsList) {
+        LOGGER.info("Create assignments map");
```

Comment:
If you can log something from btsList, better is to debug when we know what was exact argument when application fails.